docs/services/azure-services-guide.md

```diff
@@ -0,0 +1,625 @@
+kubectl create -f azure-infra-dev.yaml.yaml
+```
+
+* You should see the following output:
+```bash
+namespace/azure-infra-dev.yaml created
+```
+
+#### Azure Provider
+It is essential to make sure that the Azure Service Principal is configured with
+all permissions outlined in the [provider
+guide](../cloud-providers/azure/azure-provider.md).
+
+Using Azure Service Principal `crossplane-azure-provider-key.json`:
+* Generate BASE64ENCODED_AZURE_PROVIDER_CREDS encoded value:
+```bash
+export BASE64ENCODED_AZURE_PROVIDER_CREDS=$(base64 crossplane-azure-provider-key.json | tr -d "\n")
+```
+
+* Define an Azure `Provider` and `Secret` in `azure-provider.yaml`:
+```yaml
+cat > azure-provider.yaml <<EOF
+---
+# Azure Admin service account secret - used by Azure Provider
+apiVersion: v1
+kind: Secret
+metadata:
+  name: demo-provider-azure-dev
+  namespace: azure-infra-dev
+type: Opaque
+data:
+  credentials: BASE64ENCODED_AZURE_PROVIDER_CREDS
+---
+# Azure Provider with service account secret reference - used to provision resources
+apiVersion: azure.crossplane.io/v1alpha2
+kind: Provider
+metadata:
+  name: demo-azure
+  namespace: azure-infra-dev
+spec:
+  credentialsSecretRef:
+    name: demo-provider-azure-dev
+    key: credentials
+EOF
+```
+
+#### Create Provider
+* Create Azure provider:
+```bash
+sed "s/BASE64ENCODED_AZURE_PROVIDER_CREDS/$BASE64ENCODED_AZURE_PROVIDER_CREDS/g" azure-provider.yaml | kubectl create -f -
+```
+
+* Verify Azure provider was successfully registered by the crossplane
+```bash
+kubectl get providers.azure.crossplane.io -n azure-infra-dev
+kubectl get secrets -n azure-infra-dev
+```
+
+#### Cloud-Specific Resource Classes
+Cloud-specific resource classes are used to define a reusable configuration for
+a specific managed service. Wordpress requires a MySQL database, which can be
+satisfied by an [Azure Database for
+MySQL](https://azure.microsoft.com/en-us/services/mysql/) instance.
+
+* Define an Azure MySQL `SQLServerClass` in `azure-mysql-standard.yaml`:
+```yaml
+cat > azure-mysql-standard.yaml <<EOF
+---
+apiVersion: database.azure.crossplane.io/v1alpha2
+kind: SQLServerClass
+metadata:
+  name: azure-mysql-standard
+  namespace: azure-infra-dev
+specTemplate:
+  adminLoginName: myadmin
+  resourceGroupName: RESOURCE_GROUP_NAME
+  location: EAST US
+  sslEnforced: false
+  version: "5.6"
+  pricingTier:
+    tier: GeneralPurpose
+    vcores: 2
+    family: Gen5
+  storageProfile:
+    storageGB: 25
+    backupRetentionDays: 7
+    geoRedundantBackup: false
+  providerRef:
+    name: demo-azure
+    namespace: azure-infra-dev
+  reclaimPolicy: Delete
+EOF
+```
+
+* Create the `SQLServerClass`:
+```bash
+sed "s/RESOURCE_GROUP_NAME/$RESOURCE_GROUP_NAME/g" azure-mysql-standard.yaml | kubectl apply -f -
+```
+
+* You should see the following output:
+```bash
+sqlserverclass.database.azure.crossplane.io/azure-mysql-standard created
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
```

Comment:
The pattern I usually like to use for showing a command and its output is:

```
$ kubectl get cloudsqlinstanceclass -n ${INFRA_NAMESPACE}
NAME                   PROVIDER-REF   RECLAIM-POLICY   AGE
standard-cloudsql      gcp-provider   Delete           11s
```

Because that's what it looks like when people are in the terminal. Sometimes I'll put `> ` at the beginning of the stuff that's printed out if there's only one line of output, or if there's another prompt.